java/integTest/src/test/java/glide/TestUtilities.java

```diff
@@ -394,17 +396,18 @@ public static String createLongRunningLuaScript(int timeout, boolean readOnly) {
         return script.replace("$timeout", Integer.toString(timeout));
     }
 
-    public static void waitForNotBusy(BaseClient client) {
+    /**
+     * Lock test until server completes a script/function execution.
+     *
+     * @param lambda Client api reference to use for checking the server.
```

Comment:
```suggestion
     * @param lambda Client api reference to use for terminating the server process.
```

Reply:
We don't terminate the server. I'll reword if you don't mind